<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicMgr</class>
 <widget class="QDialog" name="MusicMgr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BtnAdd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnRemove">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnPreviewStart">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnPreviewStop">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOk">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="SongsList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
